plugins/BananaSlug/BananaSlugWebNodeMeasurer.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const BananaSlugAbstractNodeMeasurer = require('./BananaSlugAbstractNodeMeasurer');
+
+const DUMMY = {
+  bottom: 0,
+  height: 0,
+  left: 0,
+  right: 0,
+  scrollX: 0,
+  scrollY: 0,
+  top: 0,
+  width: 0,
+};
+
+class BananaSlugWebNodeMeasurer extends BananaSlugAbstractNodeMeasurer {
```

Comment:
certainly separating the logic is great, but composition would work just as well, right? class hierarchies tend to be harder to maintain (imo), and especially so in a non-statically typed language.

speaking of which -- could you get @flow annotations on this PR?